Sources/MapboxCoreNavigation/Resources/ca.lproj/Localizable.strings

```diff
@@ -1,5 +1,26 @@
+/* Format for speech string after completing a maneuver and starting a new step; 1 = distance */
+"CONTINUE" = "Continua per %@";
+
+/* Format for speech string after completing a maneuver and starting a new step; 1 = way name; 2 = distance */
+"CONTINUE_ON_ROAD" = "Continua a %1$@ per %2$@";
+
+/* Format string for less than; 1 = duration remaining */
+"LESS_THAN" = "%@";
+
+/* Format for speech string; 1 = current instruction; 2 = the following linked instruction */
+"LINKED_UTTERANCE_FORMAT" = "%1$@ i després %2$@";
+
+/* Format for speech string; 1 = current instruction; 2 = formatted distance to the following linked instruction; 3 = that linked instruction */
+"LINKED_WITH_DISTANCE_UTTERANCE_FORMAT" = "%1$@, després a %2$@, %3$@";
+
+/* Format for speech string; 1 = way name; 2 = way route number */
+"NAME_AND_REF" = "%1$@ (%2$@)";
+
+/* Format for speech string; 1 = formatted distance; 2 = instruction */
+"WITH_DISTANCE_UTTERANCE_FORMAT" = "A %1$@, %2$@";
+
 /* Error message when an offline route request returns a response that can’t be deserialized */
-"OFFLINE_CORRUPT_DATA" = "S'ha trobat una ruta no vàlida mentre fora de línia.";
+"OFFLINE_CORRUPT_DATA" = "S’ha trobat una ruta no vàlida mentre fora de línia.";
```

Comment:
These translations came back because they were still in the source files that Transifex reads. I reran scripts/extract_localizable.sh to remove the unused strings from the source files, so they’ll be removed from the translation files the next time we pull translations after merging this PR.